### ISSUE TITLE
Auto reloading not works in PyCharm

### ISSUE THREAD

**Issue by Bobsans** [pgjones/quart#48]
When I create a startup configuration and start the server with it, after changing the code, the server stops and does not start again.

**Comment by pgjones** [comment]
I haven't used PyCharm so I'm not sure how it runs Quart, could you provide the command it uses to invoke Quart?

Also can you check you have Hypercorn >= 0.5.3 installed? 0.5.3 fixed a number of reloader issues.

**Comment by Bobsans** [comment]
I tried many ways to run it.

By default PyCharm may runs main.py as file or as module.
If run as file command looks like this:
`C:\py37-env\Scripts\python.exe D:/Python/quart-test/main.py`
run as module like this:
`C:\py37-env\Scripts\python.exe -m main`
In both cases auto reloading not works.

I tried run app with Hypercorn too.
Command in PyCharm looks like this:
`C:\Users\Bobsans\py37-restory\Scripts\python.exe -m hypercorn main:app`
and auto reload still not works.

Even else run Hypercorn from PyCharm terminal (`> hypercorn main:app`) i got same result.

Maybe problem in PyCharm. I not sure.

**Comment by Bobsans** [comment]
Output from terminal:
```
(py37-env) D:\Python\quart-test> hypercorn main:app
- Running on 127.0.0.1:8000 over http (CTRL + C to quit)
- Running on http://localhost:5000 (CTRL + C to quit)
```
(i change code)
```
(py37-env) D:\Python\quart-test> c:\py37-env\scripts\python.exe: can't open file 'main:app': [Errno 2] No such file or directory
```

**Comment by pgjones** [comment]
Could you confirm which version of Hypercorn you are using?

**Comment by Bobsans** [comment]
Hypercorn 0.5.3

**Comment by tjtaill** [comment]
I use nodemon  https://nodemon.io/ to restart quart / hypercorn when I develop works pretty well, yes it is a nodejs tool but surprisingly works well with python too

**Comment by pgjones** [comment]
I think this is fixed in master with bb6989da48080478a87d4e909b3da00fe6fe6ca4 and the 0.5-maintenance branch in a4919a853e4b9b921d5132a79cf65e343432f8fe. Could you try it? If it works I'll release 0.5.4.

**Comment by slyduda** [comment]
Hot reloading is not working for me due to a space in my path name when trying to access my conda environment.
